terraform/modules/scheduled-job/README.md

```diff
@@ -221,6 +221,7 @@ module "data_processor" {
 - `main_file` - Python file name (e.g., "main.py"), relative to `source_dir`.
 - `schedule` - Cron expression (e.g., "0 9 * * 1-5")
 - `description` - Function/job description
+- `owner` - The owner/team responsible for this scheduled job
```

Comment:
Nice!  Do we want to use some consistent way of naming the team?  Like should it be a github tag (`@owner` or `#team`), or the list of teams in ownership_data.json, or something else?